libs/cart/state/src/reducers/cart-item-entities/cart-item-entities.reducer.ts

```diff
@@ -18,14 +18,20 @@ export function daffCartItemEntitiesReducer<
 	U extends DaffCartItemInput = DaffCartItemInput,
 	V extends DaffCart = DaffCart,
 >(
-  state = daffCartItemEntitiesAdapter<T>().getInitialState({ daffState: DaffCartItemStateEnum.Default }),
+  state = daffCartItemEntitiesAdapter<T>().getInitialState(),
   action: DaffCartItemActions<T, U, V> | DaffCartActions<V>): EntityState<T> {
 	const adapter = daffCartItemEntitiesAdapter<T>();
   switch (action.type) {
     case DaffCartItemActionTypes.CartItemListSuccessAction:
-			return adapter.addAll(action.payload, state);
+			return adapter.addAll(action.payload.map(item => ({
+				...item,
+				daffState: DaffCartItemStateEnum.Default
+			})), state);
 		case DaffCartItemActionTypes.CartItemLoadSuccessAction:
-			return adapter.upsertOne(action.payload, state);
+			return adapter.upsertOne({
+				...action.payload,
+				daffState: DaffCartItemStateEnum.Default
+			}, state);
```

Comment:
Are you sure we want to reset the state on item loads?

Reply:
I think it should be ok, because the `DaffCartLoad`, `DaffCartItemLoad`, and `DaffCartItemList` actions should only be called on initial state loading. I maybe could go without it, but the initial cart items daffState would be undefined until some kind of mutating action happened to them, which could be a problem if an app dev has a `daffState === "default"` check somewhere

Reply:
> I think it should be ok, because the DaffCartLoad, DaffCartItemLoad, and DaffCartItemList actions should only be called on initial state loading

I think that largely depends on the app dev, we don't decide when this happens.

I know its more verbose, but I think `daffState: action.payload.daffState || DaffCartItemStateEnum.Default` is safer.

Reply:
daffState will never exist on a DaffCartLoadSuccess.payload.items[], because it doesn't exist in the platform. 

`daffState: action.payload.daffState || DaffCartItemStateEnum.Default` will always result in `daffState: DaffCartItemStateEnum.Default`

Reply:
oh yeah...so you would have to do something like:
```ts
adapter.upsertOne({
	...action.payload,
	daffState: state.entities[action.payload.id].daffState || DaffCartItemStateEnum.Default
}, state);
```

not great...a slightly more robust pattern that works for all default values would be something like:
```ts
const defaultStatefulCartItem: Partial<DaffStatefulCartItem> = {
  daffState: DaffCartItemStateEnum.Default,
}
...
adapter.upsertOne({
  ...defaultStatefulCartItem,
  ...state.entities[action.payload].id,
  ...action.payload,
}, state);
```

I'm not sure if its worth it, but I still don't really like the idea of resetting the state for immutable loads.

Reply:
`state.entities[action.payload.id].daffState || DaffCartItemStateEnum.Default` makes sense to me. Definitely better. Changed